packages/math-inline/src/answer-block.jsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import Static from '@pie-lib/math-toolbar/lib/mathquill/static';
```

Comment:
can we avoid pulling modules directly from the `lib` dir - instead export them? Not a fix for this pr, but as we go forward.